applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_solver.py

```diff
@@ -121,28 +146,26 @@ def ImportModelPart(self):
         print ("model reading finished")
 
     def GetMinimumBufferSize(self):
-        return 2;
+        return 0
```

Comment:
Thanks for the review Philipp! I tried with 1 and 0 and both worked, so I left it with 0 since I do not need buffer. Shall I change it back to one anyway? Is there a reason why it should be 1?

Reply:
I don't know the full details of how this is implemented, but you DO need a buffer, since you are accessing the historical database
You don't need the steps back in time, i.e. you only need a size of 1

Reply:
I agree with @philbucher. It should be 1

Reply:
Just for the record, the model part will ignore requests to set the buffer size to a size smaller than the current one. Since it starts with `buffer_size=1`, it would never be set to 0 (which explains why @inigolcanalejo could see no difference).

This is done by design, to make it easier to couple solvers (so if your solver needs 3 steps and mine only 2, we can couple them without me having to increase the "single-domain" buffer size in my solver).